packages/docs/validator-guide/celo-foundation-voting-policy.md

```diff
@@ -121,7 +126,7 @@ Past Foundation votes recipients:
 * **Cohort 2:** The Great Celo Stake Off [leaderboard](https://docs.google.com/spreadsheets/d/1Me56YkCHYmsN23gSMgDb1hZ_ezN0sTjNW4kyGbAO9vc/edit#gid=1970613133) participants at ranking 1-25 -- votes will expire on Nov 1, 2020
 
 Currently receiving Foundation votes: 
-* **Cohort 3:** [6 validator groups](https://docs.google.com/spreadsheets/d/1OkWnr6EOeFn4pIv0zxmXFNtHLmKWf_qCJOJ4iacov-A/edit?usp=sharing) -- votes will expire on Feb 1, 2021
+* **Cohort 3:** [6 validator groups](https://docs.google.com/spreadsheets/d/1OkWnr6EOeFn4pIv0zxmXFNtHLmKWf_qCJOJ4iacov-A/edit?usp=sharing) -- votes will expire on Feb 1, 2021 (Note: This will transition to Cohort 6 as soon as they're voted in)
```

Comment:
it's not guaranteed that cohort 3 will automatically transition to cohort 6 -- would suggest creating a new line for C6